internal/tui/monitor.go

```diff
@@ -108,8 +110,10 @@ func StartMonitoring(timeout time.Duration, nodeStores []*store.PgedgeNodeStore,
 	}
 }
 
-func buildMonitorTable(nodeStores []*store.PgedgeNodeStore) (*table.Table, error) {
+func buildMonitorTable(nodeStores []*store.PgedgeNodeStore, includeMattermostData bool) (*table.Table, error) {
 	data := [][]string{}
+	var schemaVersion, users, teams, channels, posts string
+
 	for _, nodeStore := range nodeStores {
```

Comment:
nit: should you nil check nodeStore.Store?

Reply:
Good question. We are safe here as long as the `err` value is checked when `nodeStores` is created. Any issues creating the stores will return an error.